src/camera-helper.ts

```diff
@@ -30,15 +30,23 @@ if (constants.SUPPORTS_MEDIA_DEVICES) {
 }
 
 export default class CameraHelper {
+  flash: FillLightMode;
+  redEyeReduction: boolean;
```

Comment:
As you don't use `redEyeReduction`, it may be better to omit it as I was looking for where you were using it ;)